src/ts/components/subtitleoverlay.ts

```diff
@@ -326,6 +327,10 @@ class ActiveSubtitleManager {
       text: event.html || ActiveSubtitleManager.generateImageTagText(event.image) || event.text,
     });
 
+    if (event.regionStyle) {
```

Comment:
As far as I know, there can be multiple cues / regions at once. How are they handled by the player? Are they resulting in different cue events?

Reply:
Yes, they result in different cues with the same time range but different `region` attribute.

Reply:
Perfect 👍